### ISSUE TITLE
Simple

### ISSUE THREAD

**Issue by GAGGZ1** [Sulagna-Dutta-Roy/GGExtensions#1342]
**Is your feature request related to a problem? Please describe.**
Users often want quick access to the current time and weather without navigating away from their browser's new tab screen.

**Describe the solution you'd like**
Develop a Chrome extension that replaces the new tab screen with the current time and weather sourced from OpenWeatherMap.org. This extension should provide users with a simple and visually appealing display of the current time and weather without any unnecessary tracking or permissions.

**Describe alternatives you've considered**
One alternative could involve integrating similar functionality into existing new tab page extensions or weather apps that users already use regularly. Additionally, users could have the option to customize the display settings, such as choosing between metric or imperial units, selecting different font styles, or customizing the layout of the time and weather information.

**Additional context**
The goal of this feature is to provide users with easy access to the current time and weather directly on their new tab screen, enhancing their browsing experience. Completion of this feature can be evaluated based on user feedback regarding the accuracy, reliability, and usability of the time and weather information provided by the extension. Additionally, monitoring any changes in user engagement or satisfaction with the platform while using the extension can help assess its impact on overall user experience and retention.

**Comment by GAGGZ1** [comment]
@Sulagna-Dutta-Roy  PLEASE ASSIGN ME THIS ISSUE

**Comment by Dark-S8uL** [comment]
@Sulagna-Dutta-Roy Please assign me this issue as @GAGGZ1 is my Linkedin friend and he advised me to work on this issue.

**Comment by Sulagna-Dutta-Roy** [comment]
@Dark-S8uL how many points you've received from my repo ?

**Comment by Dark-S8uL** [comment]
@Sulagna-Dutta-Roy till now, I have added 4 extensions in the project, in which 3 are assigned with level1 and one with level2.
which adds up to 55 points in total i think.

**Comment by Sulagna-Dutta-Roy** [comment]
Checked that it's a duplicate project because already I have received weather extension before #114 
It's not acceptable as a way of contribution: "Simple Extension" is the same as weather one. For the sake of points, as per the guidelines should check if the projects is already exists/merged. Reducing the level because it's a duplicate one @GAGGZ1 @Dark-S8uL 
Because I assigned this mistakenly and received the pull request so I am not cancelling this time. But be careful, thanks
